charts/README.md

```diff
@@ -100,6 +100,19 @@ Execute command (switch to the `root` directory of the repo, and sets the `curre
 $ kubectl config use-context host
 $ helm install karmada-scheduler-estimator -n karmada-system ./charts
 ```
+
+Edited values.yaml for karmada-descheduler
```

Comment:
I think this instruction may lead to a misunderstanding because the descheduler is a component of karmada-control-plane. It has no relationship with any member clusters so it only has to be deployed once. So this guide should be moved to `## Installing the Chart` and inform users `karmada-descheduler` is not a mandatory component.

Reply:
Thanks for advice. Would you like to check again? :)